sentry-kotlin-multiplatform/src/commonMain/kotlin/io/sentry/kotlin/multiplatform/SentryId.kt

```diff
@@ -0,0 +1,44 @@
+package io.sentry.kotlin.multiplatform
+
+class SentryId(private val sentryIdString: String) {
+    var uuidString: String = ""
+        private set
```

Comment:
I'd rather prefer to use a `UUID` type so in case `fromStringSentryId` gets an invalid string, that would not work.

Reply:
UUID is unavailable in this module but I think I can find a workaround

Reply:
Oh I missed that, then its fine, its just a wrapper on top of the other SDKs anyway